### ISSUE TITLE
doesnt play .mov files

### ISSUE THREAD

**Issue by Wildersnuth** [Geektoolkit/Dynaframe3#214]
Hey I already read about this problem so i thought if i install buster and then use dynaframe, the problem should be solved. it is not, is there something i am missing?

**Comment by Geektoolkit** [comment]
The video system is based on omxplayer. It only supports mp4 and mpeg. You'll
Have to convert them to omxplayer compatable
Files

**Comment by Gira73** [comment]
@Geektoolkit Sorry to break into the discussion, but have a similar problem. I drive a 45 sec. video mp4 but it won't display . I have saved it under Videos and Pictures, but no display of the film. Is there something I didn't get?

**Comment by Geektoolkit** [comment]
What base did you install? For this it needs to be the older raspbian buster. If you install bullseye it won't work.

**Comment by Gira73** [comment]
Ok. I have bullseye. I have to install buster, so thank you.Sendt fra min iPhone26. mai 2023 kl. 20:38 skrev Joe Farro ***@***.***>:﻿
What base did you install? For this it needs to be the older raspbian buster. If you install bullseye it won't work.

**Comment by Geektoolkit** [comment]
Yeah. Just some background because this comes up alot, and in case others find this issue/thread:
   Raspbian Bullseye is based on an updated Debian version that removed OMXPlayer, which was THE hardware accelerated player that was available for Pis.  The community was told if you want hardware acceleration, 'that is the way'.  So we built our software on it, and things were good.
   
   Then Debian (outside of the raspberry pi org) made major changes to Bullseye, and omxplayer was deprecated.  VLCPlayer became the new video player, however it doesn't support rotation, zoom and other things that OMXPlayer supported. Also its a VERY different beast, the OS shipped with lower performance, and other issues.
   
   I made the decision to stay on Buster vs. spend dev cycles basically trying to 'catch up' and do a bunch of dev work that wouldn't really give the user much difference in functionality, and would likely cause performance or other issues.  However the code is here, open source, and others can feel free to take it, shim in VLC, make that and rotation and all of the features work and have a new thing for the community.  I just don't have the time myself, or honestly, the desire. I was very bummed about the way that change was handled, and to this day, even on Dynaframe Pro where we did do the VLC work, I have issues with VLC that we never had with omxplayer.

**Comment by DiegoLosco** [comment]
hi :)
where can I found the Raspbian Buster?
is it compatible with PI 4?

**Comment by Geektoolkit** [comment]
Actually that's a good point...I dont know where you'd get Buster at this point. It looks like with the pi5 they released bookworm and pulled the download off the site. I'll have to see if I can find a download but until then, this project my be dead in the water until someone updates it to work with a non 'pi' user, VLC, and more.

**Comment by sapnho** [comment]
Buster is here: https://downloads.raspberrypi.org/raspbian/images/raspbian-2020-02-14/

But when I did a fresh install for another project on a Pi 4 last night, I got the message that the Pi 4 requires "a newer software". Also here: https://forums.raspberrypi.com/viewtopic.php?t=362498&sid=6542a2dd8b9dcd3ce45a41813a84f2d4

Buster is working fine on my Pi 4 frame in the living room (using Pi3D) but something must have changed recently.

**Comment by sapnho** [comment]
I found a more recent Buster image that works on the Pi4: https://downloads.raspberrypi.org/raspios_arm64/images/raspios_arm64-2021-05-28/

**Comment by garethsnaim** [comment]
OK but whats the deal with the dynaframe images from Patreon, I cannot get any videos to display, which I thought was like the whole point of this software?

**Comment by Geektoolkit** [comment]
The Patreon support is through Patreon and discord. Please switch video out ports and reboot, that should fix videos playing. There's a help doc that gets updated and we work on trying to make issues like that better

**Comment by garethsnaim** [comment]
Ok. I am too old for discord, the interface is well named!
